packages/components/src/spinner/styles.ts

```diff
@@ -26,6 +26,8 @@ export const StyledSpinner = styled.svg`
 	position: relative;
 	color: ${ COLORS.ui.theme };
 	overflow: visible;
+	opacity: 1;
+	background-color: transparent;
```

Comment:
This could also have been:
```
opacity: inherit;
background-color: inherit;
```
which effectively resets the styles that were set by a CSS class rule, and makes them to be inherited from a parent element. Just like if the child element never specified any `background-color`.

`inherit` is better in a sense that it doesn't express any opinion about what the background color should be.

Reply:
It would inherit the background of its parent element.

If the structure is simply `div.components-spinner > svg`, then inherit would most likely make it transparent indeed, or at least the same background color as its parent. 

But I like transparent in this case, as it is an easily readable statement: this component should always have a transparent background.

Reply:
> If the structure is simply `div.components-spinner > svg`

In the spinner case however the structure is `svg.components-spinner`, both the class name and the Emotion-generated style are on the same element. It would work.

But never mind, I just wanted to share a possible alternative solution. And in case of `background-color`, `transparent` in the end means the same thing as `inherit`, just on a different conceptual level: "inherit the color of the layer that's under this one"

Reply:
Oh you're right, my mistake. But it would still inherit the background of its parent, might have its own background set. It would be invisible unless it was a tiling image, which would be unlikely, but nevertheless, not quite as explicit as "transparent".